exercises/practice/beer-song/.meta/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "blurb": "Recite the lyrics to that beloved classic, that field-trip favorite: 99 Bottles of Beer on the Wall.",
+  "authors": [
+    "FIXME-marnen"
```

Comment:
whoops should must be `marnen` I suppose

Reply:
I'm slightly confused. Aren't you the author of this exercise? As in: haven't you been the one creating these files?

Reply:
Yes. I was thinking for some reason this was the template's author. WIll change to myself thanks

Reply:
Side note: once done, you should remove yourself from the contributors list (you can be either an author _or_ a contributor).